<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Replace</string>
  </property>
  <widget class="QLineEdit" name="findLine">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>11</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="findLbl">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Find:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>82</y>
     <width>381</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="caseSensitiveCB">
      <property name="text">
       <string>Case Sensitive</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="wholeWordsCB">
      <property name="text">
       <string>Whole Words</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="directionGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Direction</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="backwardBtn">
           <property name="text">
            <string>Backward</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">backwardForwardGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="forwardBtn">
           <property name="text">
            <string>Forward</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">backwardForwardGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="replaceLine">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>40</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="replaceLbl">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace with:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>6</y>
     <width>95</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="findBtn">
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceBtn">
      <property name="text">
       <string>Replace</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceAllBtn">
      <property name="text">
       <string>Replace all</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>findLine</zorder>
  <zorder>findLbl</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>replaceLbl</zorder>
  <zorder>replaceLine</zorder>
 </widget>
 <tabstops>
  <tabstop>findLine</tabstop>
  <tabstop>replaceLine</tabstop>
  <tabstop>findBtn</tabstop>
  <tabstop>replaceBtn</tabstop>
  <tabstop>replaceAllBtn</tabstop>
  <tabstop>caseSensitiveCB</tabstop>
  <tabstop>wholeWordsCB</tabstop>
  <tabstop>backwardBtn</tabstop>
  <tabstop>forwardBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="backwardForwardGroup"/>
 </buttongroups>
</ui>
